vir/defs/middle/ast/variable.rs

```diff
@@ -0,0 +1,7 @@
+use super::ty::Type;
```

Comment:
This file is dead code (it is not mentioned among `mod`s) and should be deleted.

Reply:
:confused: yes, done.